cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/TestCommand.java

```diff
@@ -125,6 +128,16 @@ public TestCommand(Path userDir, PrintStream outStream, PrintStream errStream, b
     @CommandLine.Option(names = "--debug", description = "start Ballerina in remote debugging mode")
     private String debugPort;
 
+    // Testerina Flags
+    @CommandLine.Option(names = {"--list-groups", "-lg"}, description = "list the groups available in the tests")
```

Comment:
Do we need `-lg`? I'm not sure if it's a common practice to use two chars as a shorter option.

Reply:
`-lg` was the short option used until 0.991.x. Therefore we are providing the same in 1.1.x

Reply:
IMO, we don't need this. We have other similar options for which we don't have a shorter versions.